core/src/main/java/ar/com/wolox/wolmo/core/permission/PermissionManager.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) Wolox S.A
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package ar.com.wolox.wolmo.core.permission
+
+import android.app.Activity
+import android.content.Context
+import android.content.pm.PackageManager
+import android.os.Handler
+import androidx.core.app.ActivityCompat
+import androidx.fragment.app.Fragment
+import androidx.core.content.ContextCompat
+import android.util.SparseArray
+
+import ar.com.wolox.wolmo.core.di.scopes.ApplicationScope
+
+import java.util.ArrayList
+
+import javax.inject.Inject
+
+/** Helper class to handle Android's runtime permissions */
+@ApplicationScope
+class PermissionManager @Inject
+constructor(private val context: Context, private val requestListeners: SparseArray<PermissionListener>) {
```

Comment:
Put constructor word on the first line and the parameters on many lines.

```
class PermissionManager @Inject constructor(
         private val context: Context, 
         private val requestListeners: SparseArray<PermissionListener>
) {
```

Reply:
done